auction-server/src/server.rs

```diff
@@ -153,6 +158,10 @@ fn setup_chain_store_svm(config_map: ConfigMap) -> HashMap<ChainId, ChainStoreSv
                 chain_id.clone(),
                 ChainStoreSvm {
                     express_relay_program_id: chain_config.express_relay_program_id,
+                    client:                   RpcClient::new_with_commitment(
+                        chain_config.rpc_addr.clone(),
+                        CommitmentConfig::processed(),
```

Comment:
this client may be used for other purposes beyond simulation, maybe makes sense to not set the commitment here, and instead set it in the simulation/sending?

Reply:
if we don't set it here, the default is finalized which is rarely helpful. We can assume processed is the default and manually set it if we want to use something else